lightspark-sdk/build.gradle.kts

```diff
@@ -48,7 +48,7 @@ kotlin {
         }
         val commonTest by getting {
             dependencies {
-                implementation(kotlin("test"))
+                implementation(kotlin("test-junit"))
```

Comment:
https://issuetracker.google.com/issues/301606728?pli=1 
AS/IntelliJ doesn't like the `kotlin("test")` dependency `Unresolved reference kotlin.test.Test`, updating as a workaround

Reply:
Oh nice, thanks for resolving that :-)